Dockerfile

```diff
@@ -21,6 +22,7 @@ RUN curl --silent -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/i
 # install node and npm
 RUN source $NVM_DIR/nvm.sh \
     && nvm install $NODE_VERSION \
+    && nvm install $LTS_VERSION \
     && nvm alias default $NODE_VERSION \
     && nvm use default
```

Comment:
Sorry how will we toggle `nvm` to use LTS?

Reply:
Looking into that; I hate to add the 15th hack in this repo, but maybe just check for branch `next`? or declare something like `LTS_BRANCHES=next` in `bin/run_tests`